examples/panel/chat-with-github/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.11
+
+COPY app.py app.py
+COPY base.py base.py
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+ENTRYPOINT ["panel", "serve", "base.py", "--address=0.0.0.0", "--port=80"]
```

Comment:
we support panel without a dockerfile now, this shouldn't be necessary

Reply:
yes I forgot to remove it. was trying with Docker initially.
[Fixed](https://github.com/ploomber/doc/pull/117/commits/02ce6ad29e26c5389484f4395199443e493c91e8)